helix-term/src/commands.rs

```diff
@@ -1791,6 +1792,29 @@ fn search_selection(cx: &mut Context) {
     cx.editor.set_status(msg);
 }
 
+fn make_search_word_bounded(cx: &mut Context) {
```

Comment:
I think this should operate on the currently selected register (cx.register) and can default to `'/'`

```suggestion
fn make_register_word_bounded(cx: &mut Context) {
    let register = cx.register.unwrap_or('/');
```